DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

```diff
@@ -86,22 +86,25 @@ struct Triggers {
 };
 
 struct Digit {
-  o2::dataformats::RangeRefComp<5> ref;
+  o2::dataformats::RangeRefComp<8> ref;
```

Comment:
What is the max number of fired channels the digit may refer to? Note that with 8 bits allocated for it, the max number of BCs you can cover is MAXINT/256 = ~16 M, i.e. if you trigger on every BC, you will reach maximum at ~0.5 s. 
In case it is critical, you can use ``o2::dataformats::RangeReference<uint32_t,uint32_t> ref``

Reply:
I was sure that this <8> is number of bits for number of channels. Otherwise why you set it <5> for ZDC?

Reply:
Yes, <X> means that max number of entries you can refer to is (0x1<<X)-1. 
ZDC stores maximum 26 channels, so 5 bits are enough for them